linkschkr_test.go

```diff
@@ -0,0 +1,69 @@
+package linkschkr
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestCrawler(t *testing.T) {
+	t.Parallel()
+
+	nWorkers := 1000 // just for fun
```

Comment:
It doesn't even necessarily make sense to put a _limit_ on the number of workers, does it? I mean, if we have 75 sites to check, we could start 75 workers. More wouldn't help us; we'd just have idle workers sitting around. And fewer would just mean the job takes longer overall.

Reply:
it is supposed to be a joke showing that 1k go routines are peanuts to the OS